test/test_sparse_csr.py

```diff
@@ -89,6 +89,9 @@ def convert_layout(mat):
             return mat
 
     if mode == "all_sparse":
+        convert_layout(t)
+        convert_layout(m)
+        convert_layout(v)
```

Comment:
These lines do nothing.

Reply:
Yes. It's just for intermediate testing.